src/main/java/com/ericsson/eiffel/remrem/semantics/clone/PrepareLocalEiffelSchemas.java

```diff
@@ -50,8 +50,9 @@ private static void cloneEiffelRepo(String repoURL, String branch, File localEif
         if (!localEiffelRepoPath.exists()) {
             try {
                 // cloning github repository by using URL and branch name into local
-                localGitRepo = Git.cloneRepository().setURI(repoURL).setBranch(branch).setDirectory(localEiffelRepoPath)
+                localGitRepo = Git.cloneRepository().setURI(repoURL).setBranch("master").setDirectory(localEiffelRepoPath)
```

Comment:
Question: Should not the branch name be set to "master" ?  The logic later uses the variable branch

Reply:
If it is a branch we can clone directly using branch name instead of master but in our scenario we are using tags also. While try to clone tags directly it is showing some exception. So I have made this to clone the master branch by default and then checkout to that particular tag or branch which user has configured in the build.gradle